### PR TITLE
[RW-8188][risk=no]   Update Recent Accessed Table Styling

### PR DESCRIPTION
**Pull request by NehaBroad**
When there are no recent resources:
Screen Shot 2022-10-06 at 12 48 46 PM

https://user-images.githubusercontent.com/34481816/194373393-85ef1492-c3a8-41fb-82b5-b41d59072ab5.mov

Locked:
Screen Shot 2022-10-06 at 1 04 31 PM

Long resource name  

Screen Shot 2022-10-07 at 11 33 22 AM

Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: ui/src/app/utils/resource-list.tsx [hunk 1]
```diff
@@ -86,36 +133,83 @@ export const ResourcesList = fp.flow(withCdrVersions())((props: Props) => {
     });
   };
 
+  const getWorkspace = (r: WorkspaceResource) => {
```
[the remark this answers is not in the document]
Comment (jmthibault79) [reply ui/src/app/utils/resource-list.tsx:136]: actually......
I think we can get rid of the Workspace Map entirely.  We can do this if we include the Workspace Name in the WorkspaceResource.  Not sure if that is worthwhile to do so.

[the remark this answers is not in the document]
Comment (jmthibault79) [reply ui/src/app/utils/resource-list.tsx:136]: https://precisionmedicineinitiative.atlassian.net/browse/RW-8977

**Review by jmthibault79 — APPROVED** [review]
Looks great, works great.

`getWorkspace()` is an existing issue, so that can be a followup PR

File: ui/src/app/utils/resource-list.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (NehaBroad) [reply ui/src/app/utils/resource-list.tsx:136]: WorkspaceMap also have the workspace object, which is used to get CDR information. I can change the key from firecloudName to namespace

[the remark this answers is not in the document]
Comment (jmthibault79) [reply ui/src/app/utils/resource-list.tsx:136]: the WorkspaceResource has the CDR Version ID, which can also be used to get CDR info :)

[the remark this answers is not in the document]
Comment (jmthibault79) [reply ui/src/app/utils/resource-list.tsx:138]: https://github.com/all-of-us/workbench/pull/7106 removes `workspaceMap` :)

**Pull request merged**